patches/server/0986-fix-secure-profile-with-proxy-online-mode.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Sat, 3 Jun 2023 21:50:50 +0200
+Subject: [PATCH] fix secure profile with proxy online mode
+
+
+diff --git a/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java b/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java
+index a7e133f3495e9132a5fdae2c24f225e7b026295a..03f895c65d3d516bfdc40ef323794d2d567ca594 100644
+--- a/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java
++++ b/src/main/java/net/minecraft/server/dedicated/DedicatedServer.java
+@@ -637,7 +637,10 @@ public class DedicatedServer extends MinecraftServer implements ServerInterface
+ 
+     @Override
+     public boolean enforceSecureProfile() {
+-        return this.getProperties().enforceSecureProfile && this.getProperties().onlineMode;
++        // Paper start - fix secure profile with proxy online mode
++        var globalConfiguration = io.papermc.paper.configuration.GlobalConfiguration.get();
```

Comment:
This globalConfiguration variable isn't needed

Reply:
What would you name the config entry for this?
And I am using this for a few months now and have no problems at all behind my velocity proxy (and without) and the chat is working as it should even with a custom chat format and (when modified) with modified messages.